docs/xks/operator-guide/architecture-and-design.md

```diff
@@ -0,0 +1,117 @@
+---
+id: architecture-and-design
+title: Architecture and design
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Overview
+
+[Xenit Kubernetes Framework](https://github.com/search?q=topic%3Axenit-kubernetes-framework+org%3AXenitAB+fork%3Atrue) (XKF) is the open source building blocks for a service Xenit AB provides customers: [Xenit Kubernetes Service](https://xenit.se/it-tjanster/kubernetes-eng/) (XKS)
+
+In the perspective of [Microsoft Cloud Adoption Framework](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/enterprise-scale/architecture) (CAF), Xenit Kubernetes Service is an enterprise-scale landing zone.
```

Comment:
In the perspective:
* In the image of?
* Base upon

Additionally should we say anything about AWS here as we support multiple platforms.

Reply:
I think perspective is the correct one, because it's only in the perspective of CAF that you talk about enterprise scale landing zones. Added info about AWS.